### ISSUE TITLE
Issue in handling ["null","boolean|Doube|Integer"] types in avro schema

### ISSUE THREAD

**Issue by kunallahiri** [ScaleUnlimited/cascading.avro#46]
https://groups.google.com/forum/#!topic/cascading-user/r6UlLgw7nFk

Faced an issue in writing data to Avro Sink using Avro schema as well as passing Cascading fields and Type. Possible issue in CascadingToAvro methodtoAvro(Object obj, Schema schema). Currently fixed it in my code project to get the AvroScheme working.

**Comment by kunallahiri** [comment]
Could be related to Issue #41 .

**Comment by kkrugler** [comment]
Hi Kunal - were you thinking of submitting a patch? Mark Castillo had emailed me previously about this.

**Comment by kunallahiri** [comment]
Yes i have made changes as per the master branch and am using in my internal environment.

**Comment by kunallahiri** [comment]
Should i test it and send a pull request for your review ?

**Comment by kkrugler** [comment]
Yes, thanks

**Comment by kkrugler** [comment]
Hi @kunallahiri - could you create a pull request? Thanks!
